_quarto.yml

```diff
@@ -0,0 +1,103 @@
+project:
+  type: book
+  output-dir: _book
+  google-analytics: "UA-115082821-1"
```

Comment:
Can we instead switch to plausible by inserting this snippet in `<head>`?

```html
<script defer data-domain="r4ds.hadley.nz" src="https://plausible.io/js/plausible.js"></script>
```

Reply:
Removed Google Analytics and used `include-in-header` from https://quarto.org/docs/reference/formats/html.html#includes. I believe that's the right approach.